titanic.ipynb

```diff
@@ -5,14 +5,14 @@
       "cell_type": "markdown",
```

Comment:
_Commented on notebook `titanic.ipynb` Cell 2 Line 5_
```python
import seaborn as sns
import matplotlib.pyplot as plt

# Load the Titanic dataset
titanic = sns.load_dataset('titanic')
```
starting a review

_Respond and view the context [here](http://localhost:3000/smith-kyle/notebook-diff-tests/pull/25)._

Reply:
a new comment